cron_tasks.go

```diff
@@ -0,0 +1,29 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type CronTasksService interface {
+	CronTasksAdd(app string, params CronTasks) (CronTasks, error)
+}
+
+var _ CronTasksService = (*Client)(nil)
+
+type Job struct {
+	Command string `json:"command"`
+	Size    string `json:"size"`
```

Comment:
The size is optional

```suggestion
	Size    string `json:"size,omitempty"`
```

Do you know what `omtiempty` does?

Reply:
I do :)